adapters/vrtcal/vrtcaltest/exemplary/web-simple-video.json

```diff
@@ -0,0 +1,98 @@
+{
+  "mockBidRequest": {
+    "id": "test-request-id",
+    "imp": [
+      {
+        "id": "test-imp-id",
+        "video": {
+                "mimes": ["video/mp4"],
+                "protocols": [2, 5],
+                "w": 300,
+                "h": 250
+	},
+        "ext": {
+          "bidder": {
+            "Just_an_unused_vrtcal_param": "unused_data_for_this"
+          }
+        }
+      }
+    ],
+    "site": {
+      "id": "fake-site-id",
+      "page": "http://www.vrtcal.com"
+    }
+  },
+  "httpCalls": [
+    {
+      "expectedRequest": {
+        "uri": "http://rtb.vrtcal.com/bidder_prebid.vap?ssp=1804",
+        "body": {
+          "id": "test-request-id",
+          "imp": [
+            {
+              "id": "test-imp-id",
+	      "video": {
+		"mimes": ["video/mp4"],
+		"protocols": [2, 5],
+		"w": 300,
+		"h": 250
+	      },
+
+	      "ext": {
+		  "bidder": {
+		    "Just_an_unused_vrtcal_param": "unused_data_for_this"
+		  }
+	     }
+            }
+          ],
+          "site": {
+            "id": "fake-site-id",
+	    "page": "http://www.vrtcal.com"
+          }
+        }
+      },
+      "mockResponse": {
+        "status": 200,
+        "body": {
+          "id": "test-request-id",
```

Comment:
Same here. The spaces in yellow are tabs, can we switch them to spaces?
![image](https://user-images.githubusercontent.com/15863840/176758381-fc22ab0a-8153-413a-a20c-104e4e9a43f1.png)